modules/howtos/pages/managing-connections.adoc

```diff
@@ -111,11 +112,34 @@ NOTE: Any TLS certificates must be set up at the point where the connections are
 [#ssl]
 == Secure Connections
 
-Couchbase Server Enterprise Edition supports full encryption of client-side traffic using Transport Layer Security (TLS).
+Couchbase Server Enterprise Edition and Couchbase Capella support full encryption of client-side traffic using Transport Layer Security (TLS).
 That includes key-value type operations, queries, and configuration communication.
-Make sure you have the Enterprise Edition of Couchbase Server before proceeding with configuring encryption on the client side.
+Make sure you have the Enterprise Edition of Couchbase Server, or a Couchbase Capella account, before proceeding with configuring encryption on the client side.
 
-To configure encryption for the Ruby SDK:
+[{tabs}]
+====
+Couchbase Capella::
++
+--
+The Ruby SDK bundles Capella’s standard root certificate by default.
+This means you don’t need any additional configuration to enable TLS -- simply use `couchbases://` in your connection string.
+
+NOTE: Capella's root certificate is *not* signed by a well known CA (Certificate Authority).
+However, as the certificate is bundled with the SDK, it is trusted by default.
+--
+
+Couchbase Server::
++
+--
+As of SDK 3.4, if you connect to a Couchbase Server cluster with a root certificate issued by a trusted CA (Certificate Authority), you no longer need to configure the `trust_certificate` path in your connection string.
+
+The cluster's root certificate just needs to be issued by a CA whose certificate is in your system trust store.
+This includes well known CAs (e.g., GoDaddy, Verisign, etc...), plus any other CA certificates that you wish to add.
+
+IMPORTANT: The SDK uses an https://github.com/ruby/openssl[OpenSSL wrapper] for TLS functionality, which loads your system's certificate store by default.
+If your system does not have a default root CA store, it will use the https://ruby.github.io/openssl/OpenSSL/X509/Store.html[Mozilla CA] certificate store.
+
+You can still provide a certificate explicitly if necessary:
```

Comment:
@avsej or @DemetrisChr, could you have a look at this when you have a minute?

Just want to make sure this is correct!

Reply:
I think I would replace "well known CAs" with "publicly trusted CAs" (not sure about this though).

We are not using the Ruby OpenSSL wrapper, the functionality is implemented in the C++ SDK core, where we retrieve the platform's root CA store in a similar way to the Ruby OpenSSL wrapper. @avsej might be able to give more details on this.